src/utils/errors.ts

```diff
@@ -4,6 +4,7 @@ export class CheckError extends Error {
   constructor(data: any) {
     super(data);
     this.name = 'CheckError';
+    this.message = 'CheckError';
```

Comment:
ton message pourrait être plus clair et dire explicitement quelle valeur a fail to check